src/main/java/com/alkemy/ong/model/Category.java

```diff
@@ -0,0 +1,25 @@
+package com.alkemy.ong.model;
+
+import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+import java.time.Instant;
+
+@Data
+@Entity
+@Table(name = "categories")
+@SQLDelete(sql = "UPDATE categories SET deleted = true WHERE id = ?")
+public class Category {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+    @Column(nullable = false)
```

Comment:
Te falta agregarle el mensaje de error que deberia mostrar si  esta en null, ejemplo: @NotNull(message = "name can't be null")

Reply:
Corregido